ProcessOptimizer/samplers/doe/doe_transform.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from sklearn.preprocessing import MinMaxScaler
+
+from ProcessOptimizer.space import normalize_dimensions
+
+
+def doe_to_real_space(design, factor_space, corner_points=None):
+    """
+    Transform the design into real space
+
+    Parameters:
+    -----------
+    design: np.ndarray
+        The design to transform.
+    factor_space: Space object from ProcessOptimizer
+        The space object used to transform the design.
+    corner_points: np.ndarray or 2D list, optional
+        The corner points of the design space. If not provided, the design
+        will be transformed under the assumption that min and max values of
+        each dimension in the design space are represented in the design
+        points.
+
+        Example: np.array([[-1, -1], [1, 1]])
+
+    Returns:
+    --------
+    np.ndarray
+        The design points in real space.
+    """
+
+    # This ensure that the transformation works no matter how the design is
+    # expressed
+    # E.g., with values from 0 to 1 or from -1 to 1.
+    scaler = MinMaxScaler(feature_range=(0., 1.))
+
+    if corner_points is not None:
+        corner_points = np.asarray(corner_points)
+        scaler.fit_transform(corner_points)
```

Comment:
```suggestion
        scaler.fit(corner_points)
```
Since we aren't using the transformed data, fit is clearer.